nilearn/plotting/tests/test_surf_plotting.py

```diff
@@ -360,6 +365,9 @@ def test_plot_surf_roi():
     # plot roi list
     plot_surf_roi(mesh, roi_map=[roi1, roi2])
 
+    # plot to axes
+    plot_surf_roi(mesh, roi_map=roi1, ax=plt.gca())
```

Comment:
When you give plt.gca() to axes. Does this meant to be as 3d projection ? I think this way we don't have attribute 'view_init'. Isn't it ?